test/activity_test.rb

```diff
@@ -173,12 +173,12 @@ class ActivityTest < Minitest::Spec
       signal, (ctx, _) = activity.([{seq: [], a: false}])
 
       _(signal.inspect).must_equal  %{#<Trailblazer::Activity::End semantic=:new>}
-      CU.inspect(ctx).must_equal     %{{:seq=>[:a], :a=>false}}
+      assert_equal CU.inspect(ctx), %{{:seq=>[:a], :a=>false}}
```

Comment:
While you're at it, could you change `%{}` to `%()`? :green_heart:

Reply:
Will do in the next PR.